hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/metrics/OzoneManagerDoubleBufferMetrics.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.om.ratis.metrics;
+
+import org.apache.hadoop.metrics2.MetricsSystem;
+import org.apache.hadoop.metrics2.annotation.Metric;
+import org.apache.hadoop.metrics2.lib.DefaultMetricsSystem;
+import org.apache.hadoop.metrics2.lib.MutableCounterLong;
+
+/**
+ * Class which maintains metrics related to OzoneManager DoubleBuffer.
+ */
+public class OzoneManagerDoubleBufferMetrics {
+
+  private static final String SOURCE_NAME =
+      OzoneManagerDoubleBufferMetrics.class.getSimpleName();
+
+  @Metric(about = "Total Number of flush iterations happened in " +
+      "OzoneManagerDoubleBuffer.")
+  private MutableCounterLong totalNumOfFlushIterations;
```

Comment:
NIT: Can we rename it to numOfFlushOperations. Iteration gives the impression that we iterate through a list.

Reply:
Named this as totalNumOfFlushIterations because we have one more metric which says max maxNumberOfTransactionsFlushedInOneIteration.

Reply:
maxNumberOfTransactionsFlushedInOneIteration seems correct and it is clear what it is implying.

Reply:
Done.